core/flyout_base.js

```diff
@@ -932,3 +949,44 @@ Blockly.Flyout.prototype.onBlocklyAction = function(action) {
   var cursor = this.workspace_.getCursor();
   return cursor.onBlocklyAction(action);
 };
+
```

Comment:
Since we're talking about interfaces, this is an `abstract` class. We refer to Blockly.Flyout everywhere in WorkspaceSvg / Toolbox as the abstract Blockly.Flyout, but we never actually instantiate a Blockly.Flyout, it's always either Vertical or Horizontal. These methods need to be implemented by the subclass for everything to function properly (ie: we make assumptions in other parts of Blockly that these methods exist on a Flyout).

Which leads me to, we should probably mark this class as `@abstract`. Will look into that.

Reply:
Done, made Blockly.Flyout abstract.